pkg/kubelet/kubelet_pods.go

```diff
@@ -435,6 +435,21 @@ func (kl *Kubelet) GetPodCgroupParent(pod *v1.Pod) string {
 	return cgroupParent
 }
 
+// GetHostNameForKernel gets hostname value to set in the hostname field (the nodename field of struct utsname) of the pod.
+func (kl *Kubelet) GetHostNameForKernel(hostname string, hostDomainName string, hostnameFQDN bool) (string, error) {
+	kernelhostname := hostname
+	if len(hostDomainName) > 0 && hostnameFQDN {
+		fqdnHostname := fmt.Sprintf("%s.%s", hostname, hostDomainName)
+		// FQDN has to be 63 chars to fit in hostname (specification is 64bytes which is 63 chars and the null terminating char).
+		const hostnameMaxLen = 63
```

Comment:
This constant is already in kubelet_pods.go
Can you use the current const ?

Reply:
the other one is inside the function truncatePodHostnameIfNeeded, I will make it a global variable so we can reuse it.